worldedit-core/src/main/java/com/fastasyncworldedit/core/configuration/Settings.java

```diff
@@ -192,7 +192,17 @@
                 "List of nbt tags to strip from blocks, e.g. Items",
         })
         public List<String> STRIP_NBT = new ArrayList<>();
-
+        @Comment({
+                "If the disallowed blocks listed in config-legacy.yml should be disallowed in all edits,",
+                "not just where blocks patterns are used.",
+                " - Can prevent blocks being pasted from clipboards, etc.",
+                " - If fast-placement is disabled, this may cause edits to be slower."
+        })
+        public boolean UNIVERSAL_DISALLOWED_BLOCKS = true;
```

Comment:
Instead/on top of of making WorldEdit's list global, I'd (rather) vouch for #39 too

Reply:
Yeah just noticed that issue and implementing atm